doc/nrf/libraries/bluetooth_services/rpc.rst

```diff
@@ -76,3 +80,19 @@ Then, you can invoke build command like this:
    :class: highlight
 
    west build -b *board* -- -DCONFIG_OVERLAY=my_overlay_file.conf
+
+API documentation
```

Comment:
I added new APIs to the note. I think that it's safer to list APIs that we support (the list is not long).

As peknis suggested, we can have a link to the Zephyr's API in this section, so I will move list from note to this section, because we don't need duplication.